book/src/configuration.md

```diff
@@ -125,3 +127,11 @@ first escaper configured and ending with the default escapers for HTML
 (no escaping; `md`, `yml`, `none`, `txt`, and the empty string). Note that
 this means you can also define other escapers that match different extensions
 to the same escaper.
+
+As an example, we want `.js` files to be treated like "txt" files. To do so:
+
+```toml
+[[escaper]]
+path = "js"
```

Comment:
At the end of the week, my brain is a mush right now, so maybe I'm misunderstanding something, but this does not look right.

Reply:
It's how I use it though. XD